kafka-producer/src/main/resources/base.yaml

```diff
@@ -1,30 +1,11 @@
 haystack:
```

Comment:
Is this config file still required?

Reply:
yes , it is required to support backward compatibility.